Content.Shared/Mobs/Systems/MobStateSystem.Subscribers.cs

```diff
@@ -102,7 +102,9 @@ private void OnStateEnteredSubscribers(EntityUid target, MobStateComponent compo
         switch (state)
         {
             case MobState.Alive:
-                _standing.Stand(target);
+                if (!TryComp(target, out BuckleComponent? buckle)
+                    || !buckle.Buckled)    // DeltaV - Stop buckled mobs from standing up.
```

Comment:
```suggestion
                    || !buckle.Buckled) // DeltaV - Stop buckled mobs from standing up.
```
Again, one space before, one space after the //